index.html

```diff
@@ -122,6 +122,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+        <!-- ________ *AnisaR17* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Anisa R</p>
+          <p class="contact">
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/anisar17" target="_blank">anisar17</a>
+          </p>
+          <p class="about">Aspiring software engineer, current CS student learning how to contribute to open source softwares</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.freecodecamp.org" target="freeccodecamp" title="First Resource">Free Code Camp</a>
```

Comment:
The `target` attribute in an `<a>` tag can only take specific values, as it tells the browser where to open the link at (ie: what target to "aim" its link at):

* `_self`
* `_blank`
* `_parent`
* `_top`
* `_unfencedTop`

Here's some more information from MDN about [how `target` works on an `<a>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) that might help more.

It looks like you did this with all three of your resource links, can you be sure to update all three?